Loops/Piglet/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+
+namespace Test
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //int [,] table = new int[3,4];
+            //for (var t = 0; t < 3; ++t)
+            //{
+            //    for (var i = 0; i < 4; ++i)
+            //    {
+            //        table[t, i] = (t * 4) + i + 1;
+            //        Console.Write(table[t, i] + " ");
+            //    }
+            //
```

Comment:
?

Reply:
Es pats tikko pabrīnijos, jo tas nav mans kods. Piglet mapē, ko noklonēju no Syllabus, bija iekšā šis Test fails, kuru Tu pēc tam tikai izņēmi.